src/server/database/sentences.ts

```diff
@@ -81,7 +81,7 @@ export default class Sentences {
 
     // When getting new sentences we need to fetch a larger pool and shuffle it to make it less
     // likely that different users requesting at the same time get the same data
-    SHUFFLE_SIZE = 5000;
+    SHUFFLE_SIZE = 500;
```

Comment:
This was done to increase performance. The random sort applied is quite heavy but fine for normal traffic. During grunnskolakeppni, I lowered it to reduce stress on the server. 500 Also seems fine now, I had a large shuffle size inititally because of my misconception that we wanted as many different sentences uttered as possible, with as little overlap as possible. 

But since overlap is fine and even encouraged, 500 should be good enough.

Reply:
Perhaps the comment should be changed to reflect these difference goals then